### PR TITLE
Grammatical Fixes in DiskOS's help files.

### PR DESCRIPTION
**Pull request by ghost**
**Type:** Typo Fix

**Description:** This revises all of the DiskOS help files, making the grammar slightly better.

**Modified Section:** DiskOS

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Oh shoot. I should probably fix this:
`There's a public github repository for LIKO-12 disk: https://ramilego4game.github.io/LIKO-12-Disks`

The link is obviously dead. What should I replace it with?

**Comment by boredom101** [comment]
You should replace it with [https://liko-12.github.io/Disks/](https://liko-12.github.io/Disks/).
Thank you.

**Review by boredom101 — DISMISSED** [review]
This looks very good, I appreciate you doing the tedious but important task of correcting grammar.

**Comment by ghost** [comment]
Thanks for the feedback! If I catch anything else, I'll make sure to update this.

**Review by ghost — COMMENTED** [review]
Ok, everything should be perfect now. Enjoy!

File: OS/DiskOS/Help/API_2
Comment (pablomayobre) [inline OS/DiskOS/Help/API_2]: Backticks are used to properly format code in Markdown (which LIKO-12 uses for docs)

\`local\` becomes `local`

Honestly I'm not sure if this should remain with backticks (which would translate directly to markdown and later to the JSON format that will be used for the [API docs](https://github.com/LIKO-12/API-Documentation)) or if we want to replace it to just ticks which display correctly in LIKO-12.

I'd prefer the later since the docs will probably need to be rewritten to fit the JSON format anyway.

@boredom101 what do you think?

File: OS/DiskOS/Help/Tips
Comment (pablomayobre) [inline OS/DiskOS/Help/Tips]: This link should be `https://github.com/LIKO-12/Shaders`

File: OS/DiskOS/Help/Tips
Comment (pablomayobre) [inline OS/DiskOS/Help/Tips]: And this one should be `https://github.com/LIKO-12/LIKO-12`

File: OS/DiskOS/Help/Tips
Comment (pablomayobre) [inline OS/DiskOS/Help/Tips]: This is now available at `https://liko-12.github.io/Nightly/`

**Review by pablomayobre — APPROVED** [review]
I approve this PR, I'll wait for the comments on whether to use backticks or ticks and after that's done I'll merge even if the links aren't fixed by then (I can just commit those later)

**Review by pablomayobre — CHANGES_REQUESTED** [review]
Okey, I asked Rami and the backticks are used to highlight the words, so we want to use backticks for the highlighted parts and not ticks...

**Comment by Cheezed** [comment]
Could you elaborate? I am willing to fix whatever is necessary.

**Comment by ghost** [comment]
Oh, ok. That’s understandable. I’ll revert it back.

**Review by pablomayobre — APPROVED** [review]

**Pull request merged**
